src/style.css

```
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  margin: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  list-style: none;
+  align-items: center;
+  padding: 23px 0;
+}
+
+.nav__link {
+  margin-right: 20px;
+  font-size: 12px;
+}
+
+.link {
+  padding: 23px 0;
+  color: #000;
+  text-decoration: none;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  display: block;
+  height: 4px;
+  width: 37px;

```

What if it will be link with larger word? Will you calculate it for everyone?
```suggestion
  width: 100%;

```